generator/source/faq/consensus.rst

```diff
@@ -219,6 +219,11 @@ Blockchain achieves fault tolerance by having its state (data) completely duplic
 Distributing nodes on virtual machines sharing the same host does nothing to guard against hardware faults.
 Distributing nodes at the same site does not protect against site outages.
 
+Can I restrict what validator nodes win consensus?
+-------------------------
+No. Every peer node validates blocks and every peer node can publish a block.
+One can write your own plugin consensus module to restrict what peer nodes win. Or modify an existing consensus module to experiment.
```

Comment:
Minor nitpick:  I'd say either "You can write your own ...." or "One can write one's own...".

Reply:
OK. "You can ..."